### DISCUSSION TITLE
Tried to use javascript called hugo quiz , it runs fine on local but not when deployed to netlify

### DISCUSSION THREAD

**Discussion by dualbios4** [thuliteio/doks#862] [Q&A] [answered]
I want to put this in the dock 
`https://github.com/bonartm/hugo-quiz`
I put js in `/layout/partials/head/custom-head.html`
and added shortcodes named quizdown.html in `/layout/partials/shortcodes/quizdown.html`

what i missed? because it runs fine on local but not when deployed to netlify

**Comment by h-enk** [comment, marked as answer]
See https://github.com/h-enk/doks/discussions/312#discussioncomment-834202
